docs/en/reference/architecture.rst

```diff
@@ -85,6 +85,9 @@ be any regular PHP class observing the following restrictions:
    Also consider implementing
    `Serializable <https://php.net/manual/en/class.serializable.php>`_
    instead.
+-  PHP 7.4 introduces the new magic method ``__unserialize``, which changes the execution
+   priority between ``__wakeup`` and itself when used. This can cause unexpected behaviour in
+   an Entity.
```

Comment:
I's simply apply the same warning as for `__wakeup()` because effectively `__unserialize()` replaces `__wakeup()` if it's present.

Also, we should remove the recommendation to implement `Serializable` because that interface is considered deprecated. And `cookbook/implementing-wakeup-or-clone` might need adjustments as well.

Reply:
I agree. `__unserialize()` isn't something that can be added lightly to an entity, but should we discourage from using it as long as ORM can't use this method in entities? I expect that a future version will deprecate `__wakeup()` in favour of `__unserialize()`.